docs/src/docs/asciidoc/user/index.adoc

```diff
@@ -43,6 +43,7 @@ Each topic below corresponds to a menu item at the left.
 |===
 | Topic | Description
 
+|link:expiry.html[Expiry]|Data freshness in Ehcache 3
```

Comment:
I would add the word `expiry` or `expiration` to this description, and drop the ehcache 3 reference. Something along the lines of `Data freshness and expiration`.